reporter/kafka/kafka.go

```diff
@@ -30,15 +32,31 @@ import (
 // defaultKafkaTopic sets the standard Kafka topic our Reporter will publish
 // on. The default topic for zipkin-receiver-kafka is "zipkin", see:
 // https://github.com/openzipkin/zipkin/tree/master/zipkin-receiver-kafka
-const defaultKafkaTopic = "zipkin"
+
+// defaults
+const (
+	defaultBatchInterval = time.Second * 1 // BatchInterval in seconds
+	defaultBatchSize     = 100
+	defaultMaxBacklog    = 1000
```

Comment:
@jeqo are these good defaults?

Reply:
iiuc we are batching 100, and max size is 1000 bytes (?). 

We have these defaults in java reporter: https://github.com/openzipkin/zipkin-reporter-java/blob/3f466e56012384ee685dd5cc91012129d312289b/kafka/src/main/java/zipkin2/reporter/kafka/KafkaSender.java#L81-L90 

max request size is 1MB (default in kafka producer) and but somewhere we've discuss to reduce it to have and avoid too big batches, so 500KB should be a good default.

Make sure to align producer properties to reporter ones, similar to https://github.com/openzipkin/zipkin-reporter-java/blob/3f466e56012384ee685dd5cc91012129d312289b/kafka/src/main/java/zipkin2/reporter/kafka/KafkaSender.java#L131-L140

Reply:
@jeqo similar to http reporter:
When batch size reaches BatchSize, a collect will be triggered.
When batch size reaches MaxBacklog, spans from the beginning of the batch will be disposed.
So, the real batch size is dynamic, between 100 and 1000 (Exclude clock-triggered).
Assuming that the average span size is 500 bytes, so max size is 500KB and 1000 is a good default. The average span size is the thing we need to focus on.

Max request size is also 1MB  in sarama kafka producer: https://github.com/Shopify/sarama/blob/master/config.go#L411